app/controllers/alchemy/api/nodes_controller.rb

```diff
@@ -20,6 +20,7 @@ def index
     def move
       target_parent_node = Node.find(params[:target_parent_id])
       @node.move_to_child_with_index(target_parent_node, params[:new_position])
+      target_parent_node.touch
```

Comment:
There is a after_move callback in awesome nested set that you can use in the model instead.